examples/deadline/All-In-AWS-Infrastructure-SEP/ts/README.md

```diff
@@ -0,0 +1,62 @@
+# RFDK Sample Application - Deadline - Typescript
+
+## Overview
+[Back to overview](../README.md)
+
+## Instructions
+
+---
+**NOTE**
+
+These instructions assume that your working directory is `examples/deadline/All-In-AWS-Infrastructure-SEP/ts/` relative to the root of the RFDK package.
+
+---
+1. This sample app on the mainline branch may contain features that have not yet been officially released, and may not be available in the aws-rfdk package installed through npm from npmjs. To work from an example of the latest release, please switch to the release branch. If you would like to try out unreleased features, you can stay on mainline and follow the instructions for building, packing, and installing the aws-rfdk from your local repository.
+2. Install the dependencies of the sample app:
+
+    ```
+    yarn install
+    ```
+3.  If working on the `release` branch, this step can be skipped. If working on `mainline`, navigate to the base directory where the build and packaging scripts are, then run them and install the result over top of the `aws-rfdk` version that was installed in the previous step:
```

Comment:
We can simplify this down like in here: https://github.com/aws/aws-rfdk/blob/2acb7c4b48d09f211efeb06127b9f3ecea7478d1/examples/deadline/All-In-AWS-Infrastructure-Basic/ts/README.md

We don't need to npm install the local package since we point to the local package location as a reference in the `tsconfig.json` already. Running `build.sh` before running `npm run build` in the example will be enough to get it working and we have an issue to avoid having to do that too https://github.com/aws/aws-rfdk/issues/218

Sorry if  I gave you bad instructions on my last review, I thought I linked you to the readme example I shared here, but maybe it was the old instructions that did suggest to do the `npm install` like this.